src/main/resources/application.properties

```diff
@@ -200,7 +200,7 @@ feature.toggle.secure_doc_enabled=${SECURE_DOC_ENABLED:false}
 feature.toggle.intervener_enabled=${FEATURE_INTERVENER_ENABLED:false}
 feature.toggle.case_file_view_enabled=${FEATURE_CFV_ENABLED:true}
 feature.toggle.express_pilot_enabled=${EXPRESS_PILOT_ENABLED:false}
-feature.toggle.manage_hearing_enabled=${MANAGE_HEARING_ENABLED:false}
+feature.toggle.manage_hearing_enabled=${MANAGE_HEARING_ENABLED:true}
```

Comment:
REVERT ME

Reply:
is this just for the testing in the preview environment?
if so an alternative is to set it here in your PR
https://github.com/hmcts/finrem-ccd-definitions/blob/8aaaa84bde1cb4ec89c174f5787bdf7ae962a0df/charts/finrem-ccd-definitions/values.preview.template.yaml#L61